File: linear_search.java
import java.util.*;
public class Linear_Search {
    public static void main(String[] args){
        System.out.println("Enter the searching elemnt");
        Scanner sc=new Scanner(System.in);
        int key=sc.nextInt();
        int temp=0;
        int arr[]={2,4,5,6,8,7,9};
        for(int i=0;i<arr.length;i++){
            if(arr[i]==key){            // in case of string use (arr[i].equals(key))
                System.out.println("Element is present at "+i+" position");
                temp=temp+1;
            }
        }
        if(temp==0){
            System.out.println("Element not found");
        }

    }
}

File: linear_search.cpp
#include<iostream>

using namespace std; 
bool linear_search(int arr[],int size,int key){
    for(int i=0;i<size;i++){
        if(arr[i]==key){
            return 1;
        }
    }
    return 0;
}

int main(){
    int arr[5]={3,5,7,8,6};
    int key; //want to search
    cout<<"enter thr element you want to search"<<endl;
    cin>>key;
    bool found=linear_search(arr,5,key);
    if(found){
        cout<<"element is Present"<<endl;
        
    }
    else{
        cout<<"element is not present"<<endl;
    }
    return 0;
}   